src/style.css

```diff
@@ -1 +1,30 @@
-/* add styles here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use `*` selector (it impacts performance). Set styles only for elements that require them. Zeroing out your margins, paddings or other styles with `*` is still inefficient for the browser.